### ISSUE TITLE
update logo

### ISSUE THREAD

**Issue by afomi** [bitcoin-sv/bitcoin-sv#304]
Current logo assets:

![](https://github.com/bitcoin-sv/bitcoin-sv/blob/master/share/pixmaps/bitcoin64.png)

Please consider an open-source license for the updated logo assets as well. Thanks!

**Comment by mickmight** [comment]
Good idea. Thanks for raising. I'll ask the BSV Association if they would like to update these png files.
